Dockerfile

```diff
@@ -1,5 +1,8 @@
-FROM ruby:3.3.0
+FROM ruby:3.3.0 AS base
 ARG BUILD_TAG=local
+ARG WCA_REGISTRATIONS_URL
+ARG WCA_REGISTRATIONS_POLL_URL
```

Comment:
Why do you specifically need these two?
`ROOT_URL` kinda makes sense, but why the microservice URLs and not other env variables?

Reply:
because these are needed in the webpacker compilation process

Reply:
see `application.js.erb`